src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,118 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CalculatorTest {
+    private static final double DELTA = 0.0001;
+    private static Calculator calculator;
+
+    @BeforeAll
+    static void beforeAll() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    void additionTwoElements_Ok() {
```

Comment:
```suggestion
    void calculate_additionTwoElements_ok() {
```
the convention is <methodUnderTest>_<state>_<expectedBehavior>
check other methods as well